travis/install_protobuf.sh

```diff
@@ -11,9 +11,9 @@ if [ "$protobuf_dist" != "" ]; then
   cd $protobuf_dist
 fi
 
-wget https://github.com/google/protobuf/archive/v3.0.0-beta-1.tar.gz
-tar -xvzf v3.0.0-beta-1.tar.gz
-cd protobuf-3.0.0-beta-1
+wget https://github.com/google/protobuf/archive/v3.0.0-beta-2.tar.gz
```

Comment:
So I can confirm now that there is an error while building protobuf-beta-1 on OS X. I'm not really sure what to do now, do we have to wait for a grpc update?
Shouldn't this be mentioned in the docs to update the protobuf version on mac as a temporary workaround? Or update the install_protobuf.sh script to build beta-2, if run on OS X? I suppose it shouldn't be a problem to have beta-2 on OS X, because vitess would be only used as a dev dependency there, not a real production instance.